packages/virglrenderer-android/0009-disable-termux-detection.patch.beforehostbuild

```diff
@@ -0,0 +1,12 @@
+diff -ur orig/src/mesa/util/detect_os.h mod/src/mesa/util/detect_os.h
+--- orig/src/mesa/util/detect_os.h	2025-04-02 07:24:35.000000000 -0500
++++ mod/src/mesa/util/detect_os.h	2025-08-02 12:57:08.800000000 -0500
+@@ -24,7 +24,7 @@
+  * Android defines __linux__, so DETECT_OS_LINUX and DETECT_OS_UNIX will
+  * also be defined.
+  */
+-#if defined(__ANDROID__)
++#if defined(__ANDROID__) && !defined(__TERMUX__)
```

Comment:
Just patch it to `if defined(0)`. It is not necessary to define `__TERMUX__` in `cpp_args`.